src/Http/Cookie/Cookie.php

```diff
@@ -370,7 +371,7 @@ public function withHttpOnly($httpOnly)
      * @return void
      * @throws \InvalidArgumentException
      */
-    protected function validateBool($value)
+    protected function validateBool(bool $value): void
```

Comment:
Do we need to keep this method? The type hint will raise a `TypeError` when it fails. The method body won't run unless the parameter is a `bool` now.

Reply:
Nope, with `bool` typehints added for the `$http` and `$secure` arguments this method is no longer necessary. I'll remove it.